### PR TITLE
Combine Home/Liked/Disliked fragments into AssetListFragment

### PR DESCRIPTION
**Pull request by jackstenglein**
Removes the separate HomeFragment, LikedFragment and DislikedFragment classes and has each screen instead use the AssetListFragment class. Also renames the usecase.component package to usecase.assetList, as it currently contains only items related to the AssetListFragment. Small changes are made to the HiltFragmentScenario class in order to support launching the AssetListFragment with arguments. Tests checking the behavior from the HomeFragmentTest and LikedFragmentTest classes are moved to the AssetListFragmentTest class and the previous test classes are removed.

### CODE REVIEW

File: MoviesTVSentiments/app/src/androidTest/java/com/google/moviestvsentiments/usecase/assetList/AssetListFragmentTest.java [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    @Inject
+    SentimentsDatabase database;
+
+    @Before
+    public void setUp() {
+        hiltAndroidRule.inject();
+    }
+
+    @Test
+    public void assetListFragment_startsWithEmptyMovieList() {
```
Comment (TianyiMu) [inline MoviesTVSentiments/app/src/androidTest/java/com/google/moviestvsentiments/usecase/assetList/AssetListFragmentTest.java:53]: Something you may want to consider here is parameterized testing. I think the test cases are good now, but with parameterized tests, you can get better case coverage with less duplication.

It seems to me https://github.com/Pragmatists/JUnitParams is a good option here, but I am not sure if that test runner is compatible with HiltAndroidTest.

https://drive.google.com/file/d/186vkXaHaWEdWr21eIOSdvGfQiaOmy8Uv/view?usp=sharing
https://drive.google.com/file/d/1pJwYw1oK8U86ZJg4IR3z_PI4FqVFmvQC/view?usp=sharing
Comparison: https://drive.google.com/file/d/1WPqtZFplFHbdoVIlYilGmE7f8QV-CojW/view?usp=sharing

File: MoviesTVSentiments/app/src/androidTest/java/com/google/moviestvsentiments/usecase/assetList/AssetListFragmentTest.java [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    }
+
+    private Object[] startsWithEmptyListValues() {
+        return new Object[] {
+            new Object[] {SentimentType.UNSPECIFIED, R.id.movies_list},
+            new Object[] {SentimentType.THUMBS_UP, R.id.movies_list},
+            new Object[] {SentimentType.THUMBS_DOWN, R.id.movies_list},
+            new Object[] {SentimentType.UNSPECIFIED, R.id.tvshows_list},
+            new Object[] {SentimentType.THUMBS_UP, R.id.tvshows_list},
+            new Object[] {SentimentType.THUMBS_DOWN, R.id.tvshows_list}
```
Comment (TianyiMu) [inline MoviesTVSentiments/app/src/androidTest/java/com/google/moviestvsentiments/usecase/assetList/AssetListFragmentTest.java:72]: What about just using sentiment as the param, and check both list in the test?

Comment (TianyiMu) [reply MoviesTVSentiments/app/src/androidTest/java/com/google/moviestvsentiments/usecase/assetList/AssetListFragmentTest.java:72]: Also can you put all params before their test? That can help readability. Thanks

**Comment by jackstenglein** [comment]
I've noticed that the tests are failing again, but this appears to be unrelated to the previous issue. Before, the tests were failing with incorrect values, and now the emulator is crashing. It seems to do this when it reaches the SigninActivityTest class, which is strange, as the SigninActivity has not been changed in a while. The tests still work on my local machine, and in fact, I have not seen them fail locally yet. I'm not sure yet what the reasons could be for the failures in the continuous integration.

**Comment by TianyiMu** [comment]
Looks like a ci issue. Passed when I return.

**Pull request merged**